std_msgs/Header header
  uint32 seq
  time stamp
  string frame_id
float32 angle_min
float32 angle_max
float32 ResponseDist
uint32 LaserAngle
float32 range_min
float32 range_max
float32[] ranges
float32[] ranges2
